libs/cardano-ledger-binary/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Version history for `cardano-ledger-binary`
 
+## 1.2.0.0 (unreleased)
```

Comment:
This is a minor version bump, because it is not a breaking change. Also requires a version change in the cabal file.

```suggestion
## 1.1.1.0
```


The other important part of every version bump is going and updating every affected dependency in this repo. In case of a minor bump, like this one only the packages that use the new functionality need adjustment. For example because `cardano-ledger-core` now uses `unlessDecoderVersionAtLeast` it will require the minor version update `cardano-ledger-binary >=1.1.1 && <1.2`